activitysim/core/los.py

```diff
@@ -211,6 +211,7 @@ def load_skim_info(self):
         if self.zone_system == THREE_ZONE:
             # load this here rather than in load_data as it is required during multiprocessing to size TVPBCache
             self.tap_df = pd.read_csv(config.data_file_path(self.setting('tap'), mandatory=True)).sort_values('TAP')
+            self.tap_df = pd.read_csv(config.data_file_path(self.setting('tap'), mandatory=True))
```

Comment:
This seems like a duplicate read. It should be either or, not both.

Reply:
Fixed